profile-widget/diveeventitem.cpp

```diff
@@ -158,6 +173,10 @@ void DiveEventItem::setupToolTipString()
 		/* Do we have an explicit cylinder index?  Show it. */
 		if (internalEvent->gas.index >= 0)
 			name += tr(" (cyl. %1)").arg(internalEvent->gas.index + 1);
+		icd_data icd_data;
+		if (isobaric_counterdiffusion(lastgasmix, mix, &icd_data))
+			name += tr("\nICD ΔN2/ΔHe=%1/%2=%3%").arg(icd_data.dN2 / 10).arg(-icd_data.dHe / 10).arg((-100 * icd_data.dN2 / icd_data.dHe));
```

Comment:
Not "N2"! :-)
And I also have doubts about the "delta".
@willemferguson did recently use:
&#916;N&#8322;
`&#916;N&#8322;`
But I don't know if it has to be the same here. But it has to be a "&#8322;"